includes/admin/class-wc-admin.php

```diff
@@ -88,6 +88,7 @@ public function includes() {
 
 		// Marketplace suggestions & related REST API.
 		include_once dirname( __FILE__ ) . '/marketplace-suggestions/class-wc-marketplace-suggestions.php';
+		include_once dirname( __FILE__ ) . '/marketplace-suggestions/class-wc-marketplace-updater.php';
```

Comment:
I think we don't need this – we're not using the class outside of the WC_Queue context, but it's probably loaded anyway in `WC::includes()` so this line does nothing.